src/app/core/admin/admin-entity/admin-entity.component.scss

```diff
@@ -26,4 +26,9 @@
   ::ng-deep app-view-title {
     color: colors.$muted;
   }
+
+  &.general-mode {
+    overflow-y: hidden;
+  }
```

Comment:
This is not a feasible solution as it will make parts of the UI completely unreachable on smaller windows.

Reply:
I have removed this. If we cannot figure out the correct height for the mat-tab-group , it's better to scroll the full part rather than hide completely